actiontext/lib/action_text/attribute.rb

```diff
@@ -34,6 +34,14 @@ module Attribute
       # * <tt>:strict_loading</tt> - Pass true to force strict loading. When
       #   omitted, <tt>strict_loading:</tt> will be set to the value of the
       #   <tt>strict_loading_by_default</tt> class attribute (false by default).
+      #
+      # Note: Since Action Text relies on polymorphic associations, and
+      # polymoprhic associations rely on storing class names in the database,
+      # that data must remain synchronized with the class name used by the Ruby
+      # code. When renaming classes that uses <tt>has_rich_text</tt>, make sure
```

Comment:
Minor grammar suggestion:

```suggestion
      # code. When renaming classes that use <tt>has_rich_text</tt>, make sure
```